docs/deprecations.md

```diff
@@ -0,0 +1,75 @@
+# Deprecations
+
+Sometimes a feature of Argo Workflows is deprecated.
+You should stop using a deprecated feature as it may be removed in a future minor or major release of Argo Workflows.
+
+To determine if you are using a deprecated feature the [`deprecated_feature`](metrics.md#deprecated_feature) metric can help.
+This metric will go up for each use of a deprecated feature by the workflow controller.
+This means it may go up once or many times for a single event.
+If the number is going up the feature is still in use by your system.
+If the metric is not present or no longer increasing are no longer using the monitored deprecated features.
+
+## `cronworkflow schedule`
```

Comment:
I would name these for their respective features so that they are easily correlated, and then perhaps have a "> `cronworkflow schedule` attribute" to indicate that this is what the metric displays. I think it's difficult to tell that this is referring to those specific names unless you have specifically came from the metric's page and read it in detail.

This page isn't specific to the metric though, it's just named "Deprecations" in general  as well, so it can reference it but should be its own independent page as well

Reply:
It would also be good to link to the feature page for each of these which will have more details and examples